### ISSUE TITLE
[BUG] `[&>button]:bg-opacity-90` should be valid for `migration-from-tailwind-2` rule

### ISSUE THREAD

**Issue by nghiepdev** [francoismassart/eslint-plugin-tailwindcss#333]
**Describe the bug**
The code below is currently displaying a warning
> Classname '[&>button]:bg-opacity-90' should be replaced by an opacity suffix (eg. '/90')

```jsx
      <div className="[&>button]:bg-opacity-90">
        <button className="bg-green-400">Login</button>
        <button className="bg-red-500">Register</button>
      </div>
```

**Expected behvior**
Rule `migration-from-tailwind-2` should be valid for child selector: `[&>]`, `[&_]`

**Screenshots**
<img width="1181" alt="image" src="https://github.com/francoismassart/eslint-plugin-tailwindcss/assets/4768095/f7816ef6-0a07-469b-8435-ca0ab3f7ebe0">

**Comment by kachkaev** [comment]
> Rule `migration-from-tailwind-2` should be valid for child selector: `[&>]`, `[&_]`

Why do you think so? I’d suggest that this code should be refactored to:

```
      <div>
        <button className="bg-green-400/90">Login</button>
        <button className="bg-red-500/90">Register</button>
      </div>
```
